openwisp_radius/api/permissions.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.utils.translation import gettext_lazy as _
+from ipware import get_client_ip
+from rest_framework.exceptions import APIException, PermissionDenied
+from rest_framework.permissions import BasePermission
+
+logger = logging.getLogger(__name__)
+
+
+class IsSmsVerificationEnabled(BasePermission):
+    def has_permission(self, request, view):
+        organization = getattr(view, 'organization')
+        client_ip = get_client_ip(request)[0]
+        verification = self.has_sms_verification(view, organization)
+        if not verification:
+            logger.warning(
+                f'View {view.__class__.__name__} is being accessed for organization '
+                f'{organization.name} but SMS verification is disabled for '
+                f'this organization. Client IP address: {client_ip}'
+            )
+            raise PermissionDenied(
+                _('SMS verification is not enabled for this organization')
+            )
+        return verification
+
+    def has_sms_verification(self, view, org):
+        try:
+            verification = org.radius_settings.sms_verification
+        except Exception:
```

Comment:
please use `ObjectDoesNotExist`, we don't expect any other exception, if there will be any other exception it's better to let it fail and then fix it while also adding another test case for it.